dev-compose.yaml

```diff
@@ -2,11 +2,13 @@ name: photoview
 
 services:
```

Comment:
As I see here only the API and UI services are defined, please remind me how a developer is supposed to run MariaDB or PostgreSQL DB if needed or preferred.

Reply:
I don't have a good way to launch an external db yet. I'll improve it in the future.

Reply:
let's add the 2 DB services commented out to the dev-compose file in the same way as it is done in the PROD compose example. Then if it is needed a dev can simply uncomment it and run.
Sure, there should be corresponding config lines added (also commented) to the Backend service

Reply:
I updated `dev-compose.yaml` with database, and README about how to use it.